modules/integration/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "int_logid" {
+  value = { for v in oci_logging_log.integration_log : v.display_name => v.id }
+}
+
+output "int_loggroupid" {
+  value = { for k, v in var.loggroup : v.display_name => v.id }
+}
```

Comment:
I would recommend to add an `all_attributes` output on top of these easy/quick access outputs. See [VCN module outputs](https://github.com/oracle-terraform-modules/terraform-oci-vcn/blob/main/outputs.tf) for example.

```
output "vcn_all_attributes" {
  description = "all attributes of created vcn"
  value       = { for k, v in oci_core_vcn.vcn : k => v }
}
```

This would allow:
- access to any attributes generated by the module: id and name are the obvious one, but you don't really know which attribute a module user will need
- low maintenance: the outputs be will auto-generated as the provider add new attributes
- consistency across our modules: vcn and compute already use such technique.

Reply:
Other observations about how the outputs are built:

1. using display_name as key is risky as it is not necessarily a unique value. You could potentially overwrite  entries when using `count` or `for_each `? I would rather use the ocid as the key as it is guaranteed to be unique.
2. you build the keys on a variable, I would rather build it on a module object to get the real attributes values from the API, not the initial variable assignment.

Reply:
I would not add all_attributes at this stage as there are already many outputs and if a requirement comes then i am planning to add .